### ISSUE TITLE
It doesn't seem like standard markup.

### ISSUE THREAD

**Issue by chaney-space** [StartBootstrap/startbootstrap-sb-admin-2#381]
I see wrong markup on the index.html page.
Another element is marked up between ul and li.
Has the html standard changed?

**Comment by tiesont** [comment]
No, the standard is the standard. The template is a template - you're not really supposed to just use it as-is.

I'm sure that a pull-request fixing whatever is invalid would be appreciated.
